components/Breadcrumb/src/breadcrumb.css

```diff
@@ -0,0 +1,125 @@
+.denhaag-breadcrumb {
+  --denhaag-breadcrumb-background-color: transparent;
+  color: var(--denhaag-breadcrumb-color);
+}
+
+.denhaag-breadcrumb__list {
+  block-size: 100%;
+  display: flex;
+}
+
+ol.denhaag-breadcrumb__list {
+  list-style: none;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  padding-inline-start: 0;
+}
+
+.denhaag-breadcrumb__item,
+.denhaag-breadcrumb__link {
+  align-items: center;
+  display: flex;
+}
+
+.denhaag-breadcrumb__item:not(:first-child):not(:nth-last-child(2)):not(:last-child) {
+  --denhaag-breadcrumb-background-color: var(--denhaag-breadcrumb-dots-background-color, var(--denhaag-color-grey-1));
```

Comment:
I kinda disagree with you here. 

First of all you want the CSS variable `--denhaag-breadcrumb-dots-background-color` to be set and modified, but if it's not set you want a variable fallback value which is quite common/global. Therefor you've a fallback which is dynamic and responds to other theme colors.

Reply:
For clarity, it's not that I think having a fallback is bad. The issue is that these fallbacks are using the Den Haag (proprietary) color palette tokens. A fallback token would only be valuable to other organisations because for Den Haag we already define the component tokens, but another organisation should not fill our color palette. Tokens like `denhaag-color-{green,red,blue,grey,orange,ocher}` have no meaning for other organisations as their color palette could have completely different colors. Which means using them as a fallback is useless. @Robbert any thoughts?

Reply:
@bddjong , Not sure who, but I was told that when this went to the nl-design repository the prefixes would be set to `--nl-design` instead of `--denhaag`. When that happens, doesn't that tackle the whole issue?

Reply:
@paulAcato I agree there is no need for the `--denhaag-color-grey-1` fallback: all components should only contain "semantic" design tokens such as component level tokens and common tokens regarding common patterns such as focus color. Any design decisions should strictly be separated from the component code and they should be documented in Style Dictionary, so they remain in sync with Figma where we also (intend to) use `tokens.json` files.

Having these design tokens makes the code a bit less "forkable" too, because now you not only need to rename the vendor prefix to your own prefix, but also clean up these design tokens that are basically dead code for the rest of the world.

Reply:
I'll replace my fallback values to hard values from the design.